experimental/torch_xla2/torch_xla2/ops/jtorch.py

```diff
@@ -163,4 +163,13 @@ def scaled_dot_product_attention(
     res = _tpu_flash_attention(jquery, jkey, jvalue, env)
     return env.j2t_iso(res)
 
-   return _sdpa_reference(query, key, value, attn_mask, dropout_p, is_causal, scale)
\ No newline at end of file
+   return _sdpa_reference(query, key, value, attn_mask, dropout_p, is_causal, scale)
+
+@register_function(torch.Tensor.__getitem__)
+def getitem(self, indexes):
+  if isinstance(indexes, list) and isinstance(indexes[0], int):
+    # list of int, i.e. x[[1, 2]] NOT x[1, 2] (the second would be tuple of int)
+    indexes = (indexes, )
+  elif isinstance(indexes, list):
+    indexes = tuple(indexes)
+  return self[indexes]
```

Comment:
qq: which function `self[indexes]` will call? I don't see `__getitem__` in `XLATensor2` class, does it call the `__getitem__` in `torch.Tensor` base class?

Reply:
self is of type jax.Array.